doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -144,6 +144,15 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-4-4 v2-4-3
+
+DRGN-23302: The Bluetooth subsystem may in some cases deadlock when a Bluetooth link is lost while there is data transfer ongoing.
+  When this happens, the disconnected event is never delivered to the application.
+  The issue occurs when :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` is enabled which is enabled by default on nRF5340.
+  The issue is a result of the fix applied in DRGN-21085.
+
+  **Affected platforms:** nRF5340
+
```

Comment:
Any workaround?

Reply:
Resetting the device is the only thing that resolves it, so there is no good workaround for it